source/common/http/utility.cc

```diff
@@ -73,17 +73,12 @@ void Utility::appendXff(HeaderMap& headers, const Network::Address::Instance& re
     return;
   }
 
-  HeaderString& header = headers.insertForwardedFor().value();
   const std::string& address_as_string = remote_address.ip()->addressAsString();
-  HeaderMapImpl::appendToHeader(header, address_as_string.c_str());
+  headers.appendForwardedFor(address_as_string.c_str());
 }
 
 void Utility::appendVia(HeaderMap& headers, const std::string& via) {
-  HeaderString& header = headers.insertVia().value();
-  if (!header.empty()) {
-    header.append(", ", 2);
-  }
-  header.append(via.c_str(), via.size());
+  headers.appendVia(absl::string_view(via.c_str(), via.size()), ", ");
```

Comment:
nit: maybe just kill this utility function and do this as a anonymous function inside conn_manager_utility.cc?

Reply:
Hmmmm async_client_impl also uses it, worth staying in utility?

Reply:
Wherever it lands, you can just write this as:
```
headers.appendVia(via, ", ");
```

Reply:
OK sorry I did code search but must have missed that. It's fine to keep it as a utility. I do think we should be consistent about `,` vs. `, ` as I mentioned elsewhere.